rpi_data/modules/fetch_sis_course_info.py

```diff
@@ -125,8 +125,13 @@ def run(self):
                     if info[7][-2:] == 'AM':
                         info[6] += 'AM'
                     else:
-                        col = info[6].find(':')
-                        if (int(info[6][:col]) < 8) or (int(info[6][:col]) == 12):
+                        col1 = info[6].find(':')
+                        col2 = info[7].find(':')
+                        if int(info[6][:col1]) == 12:
+                            info[6] += "PM"
+                        elif int(info[7][:col2]) == 12:
+                            info[6] += "AM"
+                        elif int(info[6][:col1]) < int(info[7][:col2]):
```

Comment:
Can we rename this and add some comments here? I'm having trouble following the logic here.
Also might be nice to alias or rename some variables here

Reply:
I can add comments, info is the name of the course information, so there can't really be much more descriptive for an all-encompassing variable, but I'll get some good comments in there